lib/manageiq/appliance_console/database_replication_standby.rb

```diff
@@ -162,8 +163,18 @@ def initialize_postgresql_disk
                                     :volume_group_name   => PostgresAdmin.volume_group_name,
                                     :filesystem_type     => PostgresAdmin.database_disk_filesystem,
                                     :logical_volume_path => PostgresAdmin.logical_volume_path).setup
+
+        # if we mounted the disk onto the postgres user's home directory, fix the permissions
```

Comment:
> postgres user's home directory

Wouldn't that be /home/postgres ?

Reply:
Nope:
```
[root@localhost ~]# su - postgres -c 'echo $HOME'
/var/lib/pgsql
```

Reply:
Also, this is just a copy of the fix I made in https://github.com/ManageIQ/manageiq-appliance_console/pull/84.

For this bug fix I didn't want to touch the internal database stuff to DRY it up.

Reply:
And also, the real fix is just the `restorecon` part, not sure why we're not having a problem with the directory permissions as well so I figured I would fix this one too.

Reply:
Could the permissions already be fixed by the above referenced #84 ?  Otherwise, I don't know why it's not complaining.  Postgresql is quite vocal about it's desire specific permissions.